### ISSUE TITLE
WASM Target break in 1.7.1

### ISSUE THREAD

**Issue by JagadishaIncture** [alexzhirkevich/compose-cupertino#76]
<img width="1670" alt="Screenshot 2024-11-18 at 7 18 05 PM" src="https://github.com/user-attachments/assets/2d35f2ca-d5cc-455c-a7ee-e591fccdc471">

Hi ,

Web Target is throwing above error while using AdaptiveWidget 

`@ExperimentalAdaptiveApi
@Composable
fun AdaptiveText(
    text:String,
    modifier: Modifier = Modifier,
    style:TextStyle
) {
    AdaptiveWidget(
        material = {
            Text(
                text=text,
                modifier=modifier,
                style=style
            )
        },
        cupertino = {
            CupertinoText(
                text=text,
                modifier=modifier,
                style=style
            )
        }
    )
}`

Config

CMP: 1.7.1
Kotlin:2.0.0
Compose-cuptertiono : 0.1.0-alpha4

**Comment by schott12521** [comment]
Probably related to https://github.com/alexzhirkevich/compose-cupertino/issues/55 - CMP 1.7.1 is known to break compose-cupertino 0.1.0-alpha04

If you're willing to dive a bit deeper, we're still working on getting a release of a compatible version out...hopefully soon

**Comment by JagadishaIncture** [comment]
Suggest me which version of CMP to use which is compatable with compose-cupertino 0.1.0-alpha04

**Comment by schott12521** [comment]
> Suggest me which version of CMP and kotlin is compatable with compose-cupertino 0.1.0-alpha04

I think you could use CMP 1.6.1 and Kotlin 1.9.23

**Comment by JagadishaIncture** [comment]
CMP 1.6.11 and Kotlin 2.0.0 combination worked. Thanks

**Comment by schott12521** [comment]
@JagadishaIncture if you'd like to, you can try out my release of this library on my GitHub. It should work with CMP 1.7.1

https://github.com/schott12521/compose-cupertino

**Comment by StefanOltmann** [comment]
Ah, great. I’m waiting for being able to update, but the dependency on this library prevented that. I’ll try your fork.

**Comment by StefanOltmann** [comment]
@schott12521 Your fork is unfortunately not available on Maven Central yet and GitHub packages requires a login.

@alexzhirkevich Can you publish a new release now that we have fixes for the new Compose version?

**Comment by schott12521** [comment]
> @schott12521 Your fork is unfortunately not available on Maven Central yet and GitHub packages requires a login.


Correct, sorry about that, I've never needed to publish to maven directly! I'll get this done in the new year.
